### ISSUE TITLE
Trailing whitespace at the end of generated lines

### ISSUE THREAD

**Issue by dinjas** [hintmedia/rspec-jumpstart#4]
Have noticed this mainly on `result = ...` lines that are generated.

For example: 

```
  # TODO: auto-generated
  describe '#invoiced?' do
    it 'invoiced?' do
      spot = described_class.new
      result = spot.invoiced?  

      expect(result).not_to be_nil
    end
  end
```

**Comment by dinjas** [comment]
I believe this whitespace may be coming from here:

https://github.com/tjchambers/rspec-kickstarter/blob/master/lib/rspec_kickstarter/erb_templates.rb#L19

here:

https://github.com/tjchambers/rspec-kickstarter/blob/master/lib/rspec_kickstarter/erb_templates.rb#L72

and here:

https://github.com/tjchambers/rspec-kickstarter/blob/master/lib/rspec_kickstarter/erb_templates.rb#L107

**Comment by tjchambers** [comment]
Thanks for your report. This has been fixed in version 1.0.6.
